Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/util/GetHashMac.java

```diff
@@ -42,6 +43,12 @@
 
     public static final String MAC_REGEX = "([0-9A-Fa-f]{2}[:-]){5}[0-9A-Fa-f]{2}";
     public static final Pattern MAC_PATTERN = Pattern.compile(MAC_REGEX);
+    public static final String[] UNIX_COMMAND = {"/sbin/ifconfig -a || /sbin/ip link"};
```

Comment:
private static

Reply:
and other public static in this class should be private

Reply:
Fixed